server/views/global/header.ejs

```diff
@@ -12,6 +12,8 @@
     <meta name="msapplication-config" content="/browserconfig.xml?v=yyx75bRMkG">
     <meta name="theme-color" content="#ffffff">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="keywords" content="<%= metatags %>">
+    <meta name="description" content="<%= metadescription %>">
```

Comment:
If like to see these wrapped in an if statement much like the GA code 
This avoids invalid html when no options are entered into the setting